subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/interna/span/SpannedStringBuilder.kt

```diff
@@ -0,0 +1,33 @@
+package com.avito.android.build_verdict.interna.span
```

Comment:
package name typo: `interna` vs `internal`

does not add as a github suggestion, because it required refactoring in IDE